examples/platforms/da15000/README.md

```diff
@@ -15,29 +15,30 @@ $ make -f examples/Makefile-da15000
 
 Execute script to prepare image for flashing:
 
-```
+```bash
 $ cd ./third_party/dialog/DialogTools
 $ ./imgprep.sh
 ```
 
 Flash Binaries:
 
 1. Open another terminal window in order to start gdb server. Write the command (Linux):
-```
+```bash
 $ JLinkGDBServer -ir -if swd -speed auto -device Cortex-M0 -logtofile on -localhostonly
 ```
-
 2. Return to previous terminal window and execute a command in order to flash the board:
-```
-$ ./cli_programmer -b uartboot.bin gdbserver write_qspi 0x0 ../../../output/bin/arm-none-eabi-ot-cli.img
+```bash
+$ ./cli_programmer -b uartboot.bin gdbserver write_qspi 0x0 ../../../output/bin/arm-none-eabi-ot-cli-ftd.img
 ```
 
 ## Interact:
 
 Board will indicate state of device according to LED blink speed.
-* 5Hz    - Leader
-* 2Hz    - Router
-* 0.5Hz  - End Device
+Frequency | Role |
```

Comment:
Previous solution was tested with https://jbt.github.io/markdown-editor. Outer pipe are optional due to specification, but maybe pipe at the end of each row cause an issue. Pipe at the beginning of each row is added in latest update.